WooCommerce/WooCommerceTests/Universal Links/Routes/PaymentsRouteTests.swift

```diff
@@ -0,0 +1,77 @@
+import XCTest
+import TestKit
+@testable import WooCommerce
+
+final class PaymentsRouteTests: XCTestCase {
+
+    var deepLinkForwarder: MockDeepLinkForwarder!
+    var featureFlagService: MockFeatureFlagService!
+    var sut: PaymentsRoute!
```

Comment:
We can make those private: 

```suggestion
    private var deepLinkForwarder: MockDeepLinkForwarder!
    private var featureFlagService: MockFeatureFlagService!
    private var sut: PaymentsRoute!
```

Reply:
👍 but if someone starts instantiating the test class to access its properties, we have other problems 😂